modules.json

```diff
@@ -159,26 +159,6 @@
             },
             "subworkflows": {
                 "nf-core": {
-                    "bam_dedup_stats_samtools_umitools": {
-                        "branch": "master",
-                        "git_sha": "f4596fe0bdc096cf53ec4497e83defdb3a94ff62",
-                        "installed_by": ["subworkflows"]
-                    },
-                    "bam_sort_stats_samtools": {
-                        "branch": "master",
-                        "git_sha": "f4596fe0bdc096cf53ec4497e83defdb3a94ff62",
-                        "installed_by": ["fastq_align_star", "subworkflows"]
-                    },
-                    "bam_stats_samtools": {
-                        "branch": "master",
-                        "git_sha": "f4596fe0bdc096cf53ec4497e83defdb3a94ff62",
-                        "installed_by": ["bam_dedup_stats_samtools_umitools", "bam_sort_stats_samtools"]
-                    },
-                    "fastq_align_star": {
-                        "branch": "master",
-                        "git_sha": "4cef5bd781be21ecb80de664d1a8c5eba980f304",
-                        "installed_by": ["subworkflows"]
-                    },
```

Comment:
these subworkflows have been removed from the modules.json file, you probably don't want that